Libraries/Components/View/ViewPropTypes.js

```diff
@@ -464,6 +464,20 @@ export type ViewProps = $ReadOnly<{|
    */
   accessibilityStates?: ?AccessibilityStates,
 
+  /**
```

Comment:
I thought we were putting these kinds of props on ViewWindows until we are ready to submit it directly to the facebook repo?

Remember not all users of react-native-windows will be using changes in this repo -- they might be using the non-forked version of RN.

And does any of these change actually affect runtime?  It looks mostly like it just affects flow types?